src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -17,33 +17,74 @@ import { hasFetchedBoAndMoData, setFetchedBoMoData } from "../../utils/update/be
 import { mapPscToBeneficialOwnerGov, mapPscToBeneficialOwnerOther, mapPscToBeneficialOwnerTypeIndividual } from "../../utils/update/psc.to.beneficial.owner.type.mapper";
 
 import { CompanyPersonsWithSignificantControl } from "@companieshouse/api-sdk-node/dist/services/company-psc/types";
+import { BeneficialOwnerIndividual } from "../../model/beneficial.owner.individual.model";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
     const appData: ApplicationData = getApplicationData(req.session);
     if (!hasFetchedBoAndMoData(appData)) {
-      await retrieveManagingOfficers(req, appData);
+      if (!appData.update) {
+        appData.update = {};
+      }
+      appData.update.review_beneficial_owners_individual = [];
+      appData.update.review_beneficial_owners_corporate = [];
+      appData.update.review_beneficial_owners_government_or_public_authority = [];
+
       await retrieveBeneficialOwners(req, appData);
+      await retrieveManagingOfficers(req, appData);
       setFetchedBoMoData(appData);
     }
+    if (appData.update?.review_beneficial_owners_individual?.length){
```

Comment:
better to !== compare

Reply:
Actually, I think need to call checkAndReviewBenifitialOwners always.  Because of need to validate the last one  put into review in case of back button.